pkgs/development/python-modules/vcard/default.nix

```
@@ -0,0 +1,35 @@
+{ buildPythonPackage
+, fetchFromGitLab
+, isPy27
+, lib
+, mock
+, python
+, python-dateutil
+, wheel
+}:
+buildPythonPackage rec {
+  pname = "vcard";
+  version = "0.15.4";
+
+  disabled = isPy27;
+
+  src = fetchFromGitLab {
+    owner = "engmark";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7GNq6PoWZgwhhpxhWOkUEpqckeSfzocex1ZGN9CTJyo=";
+  };
+
+  propagatedBuildInputs = [ python-dateutil ];
+
+  checkPhase = "${python.interpreter} -m unittest discover test";
+
+  pythonImportsCheck = [ pname ];
```

```suggestion
  pythonImportsCheck = [ "vcard" ];
```
we don't even start this